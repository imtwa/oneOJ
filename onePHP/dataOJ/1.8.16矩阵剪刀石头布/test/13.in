46 79 81
SSPSRSRRPRRRRSSRSSPRRRPPPRPSSSRPRPPRSPSPPRSPSPRSPRRPRPPPPRRRPRRSRRPRPPPPRPRSPPP
RPRRSPRPSPRRSPRRPRSSRSRSPPPSRPPRRRRRRSSPPPSPRPSRRSPPSPSPPRPPRPRPSSPSSPPRSSRRPPP
PPRRRRSSPRPRRSRRPPSRRPRSRPSRPPPPPRRSRRRRSSRRPSRSSSRPRSPPPSRSPPRSSRPPRPPSRSSPSPS
SRPRSPSSRSRRRRPPPSSRRRPRRPPSPRRRSRPRPPSSRRRSRPPPRSPRRPRRRSRRRSSRPSPPRRRPPSSRPSR
SRPRRSRPPRRRRSRRPPPPPPPSSPRPRRSPSRPSSSPRRRSSRSPRRSRPSSRSSPSSPRRPPSSRSRPSPSRRSRP
PPSSPPRSRPRPRRRPPPPSSRRSSSRSSPRPPSRRSPRPRRSRSPRRRPSRPPSRPPPRSSPSRPRRSSPRRPRPRSP
RRSSSRSPSPPRRPPSRPRRRRSPRRPSRPPRRRSSPSSSPPSPSSSSRSSRSRRPSPRSSPRRSPSRSSRPRSSSPRS
PPSSSPRPSPRRRRRPPSPSRSPPRPSRRPRRSPRRPPPRPSSRSPRSPRRSPRSSSPSPSRSRRRPPSSRPSRPRSPP
PSPPRRPRRRSRRSPRRRRSPPSPSSPPSRPRSRSRSPPSPSRPSRRRSSRSRPSRRRSRRSRSSRRSSPRPPRSRSPP
SPSSSSSRSPRSSRPRPSPRRRSRPRRSRRPSPSRPRSRSPPRRPSRRPPPRSPPPSSPPPSSRSPPRRSPRRSRPSRR
PRPPRPPRPSRSRRPSSRSPSRSPSSRPRSPRPRSPRPSSRRSSPSPRRPRSPRPPSRPPPSRSRPSPSSPRPPSPSRR
SRPRPSRSRPPPRPPRSSRRSRPPRSRRPSRSPRRRRSPRPSRRPRSPSRRPSRPRRRSPRRPRRRSSRRPSRPSRSPS
SPRSRPSSRPSPPPRRRPSSRPRRPPRPSPPRRSPPPSSPRSSPRPSSPPSPSPPPSRSPSRRPPSPPSPPSSPPPPSP
PPPSRPPPRRRSPPPRPRRPPRPRRRPRPSSRRRSPSRRPPSPSRSPSPPRPPPRRRPSRSSPSSRPSSPRSPPRPPPP
PRPRPRPPRPPSRSSRRPRSRSRRPSPRRSPRRSPSRSSSPPRSSSRSSRRRSSSSRSSSRPRSRRPRRRSSRPSPRPR
RSSPSSPPRPPSPSSRRSRRSRSPSPPPSSSPPRSPSPSRSRPPRRPRSSRRRRSPSRPPRRPPRRRPRPPPRSPRRPS
SRRPRRPPRSSPSSPPSPRSSRRRPPPSRSSRRRRRSSPSSPRSPRRSRSSRRSRRSSRSPPSSRSRPRSPRSPRRSPR
PPRSSSRPPPSPPRRRSRRPSRPSRRPPPSRPPRRSRRRPRPSPSSRSSSRRRPPRRPPPPPRPRRSPPRPSPSPSRPS
PPRRSPSPSSPRRRSSPPRRSPRPRRSRRRPRSRRSSPPRSSRPPSRRPRPSRRRRRPRRRSSRRPPPPSRRRSPPSPP
PSRRSPSSPSPPPSRPRRRPPRRPPRRSRPRPRPRSPSPRRRSRPPPPSPSSSRSRSSPPSSPRRRPPPRPSPRRPPSP
SPPPSRRPPSRSSSPPPSPRSRSRRPSSRSPPSSPSSPSRSSSSPRPPSPSSPSRSPPRRRRSSRRRPRSRRRPPPRPP
RPSSRSSSPPRSSPSPPRPRSSRRRSRPSSPRRSPPRRRSSPRPRSRSSSRRRSPPSSSRPRPRSRRPSPSSRPSRPRR
SRSRRPRPSRSPSSSRRRPRRRSSRPSSSSRRSPSPPPPRSRSPSPPSRRPPSRSRRPPSPRPPPSRPRSPPPSPRRRP
RSRSPPSRRSPRSSRSRSPPRRSSPPRSRPPPSRSPRSSRPPRPSRPSSRPPSPRSPPSPPRRSPSSPSRPSSPRPRSR
RRRRRPRSSPSSSRSSRSPSPRPRPRSRRRRSSSPSRSPRPRSSPPSPSSSRRSSPPPPPPRPSRSSPRSRPSPPRSSP
RPSSSSSRPSSSRPPSRRRRPRRSPRRPSPSPSRSSRSRRSSPRPSPSRPRSRRRSRSPSRSSPRSPPPPSRRSSSSRS
SSSSSRSRPRSSPPRRPRRSSPRPPRRSPPRPRRSPRSRSPSPSPSRPRRSSRPRSPRPSPPPSSRSRPPSSRPRPPRP
SPPPSPSPRSSRSRSSSRRPPPPSRPRPPSPSPPSSSPRPRRSSSSRRSRRRSPSSPPSSPRSRPPSPRSPPPRSRPRP
PSSPPSPRSRRRRSRSPPPSSPSSSSSSPPRRPRRSPSPPSPRPRSPPRRRSPPSPPRPRRRPRRPSRRSRRPSSRSSS
PSSRRSSSRSRPRRSRRSRPSRPSSPRRRRPPRSPSRRPPSRSRPRRPSSPPPSPSSSRPPPSRPPSSRSPSPRPSPPP
PPRRSSRSPSRPSRPSRPRRSSSPSRRSPRSSRPSRPPSRPPPSPRPRSSRSRRRSPPRPRRPSSSSPSSRRRPRPSPR
PRPRPPPPPSPRRRSPPRRRRPSSPSPRRRSRSPPSSRPRPPPPSRRSPPRPPRSRPPPSRSPRRPPRRSRRPSRRPPS
PRRRSRRRSRSPSPPRRPPPPSSPPRSRRSRSSSSRRSSSSSPSSPSSPSPRRRPRPRPSRSSPRRPRSPPSRSPSRSP
PPRRPSPPSPSSSRSSPSRPRSRSPPPRRRPPRRRPPPPSPSRSPPRSPRPRPSPRSRRPRPRPPRPSPSSPPSSSPRR
RPPPSSRSPSRPSSSPPSSPPRPSPRSPPPRSSSSSSRPSRSRRPPSPRPSPPRSRPPRSPRSRSPPPRPSRPPSRRPP
RRSPSPPPRSPSSRRPSPPSSRPPRPPSSRSSPRRSRRRRSRPRRPPRSRSPRRSRPPRRPSRSSRRRPSRPRRSSPSR
SPPSPSSPSSPRPPSPPRSRRSPRRRPSPRRRSPRSSSRSRPSPPPRSPRRRPRRPSSRSRSRPSPRRRPPSSRSSPSP
PRSPRRRRPPPRPSRSSPRRRRRSPSPPPRPPRPPSPPRSRSRRPRSRPRPSPRPSPPSRPRRPRPPSPPPSSPRPPRP
SPRSPPRSPRRSRRPSSPRRPPSRRRSRRPSRRSSPPSSSSSPSSPRSRSPPSRRSPPPSPRRSRSPPRRSSRRSSPRP
SRRRPRRPPPPPRPRPRPRRPSSPRSSPRRPRRPPPPPSSPPSRPSPRRRRPPSSRRPSRSPSSRSSRRSPRPRSSRSP
RRRSPRPPRRPRRRRRPSPSSRPRPRPSRPPSRPSPSRPSRSRRSRPSSSSPRSPRSSSPPSSRSRPSSRPPPSPRSRS
RSSPSPPPRPSPSRPRPSRPRPPSPRRPRRPRRRRPPSRPSRSSRSRSPPPPRPPRRPSPSPSSSSSSPRSSRPRSRPP
PPSRRPRRRSPSPRRPSSPPSPRSSSRRSPRPPPPSPRPSPSPSSPRPPPSPPRPRRPRPPRRRPRPSPRRRRRRRSPR
RSPRPPRSRPRRSSRPRSPSPSPSRPRRRSPPSSPRRPSRRRRPRPPSRSPRRPRSPRSPSPSPPSPRPSRSRSRSRSR
SSRRRPPSPRPPRPSRRSPRPPSSRSRPRPRRPRPSRRPSRRSRRPRPRPRRSSRPPRRRSPPPRSSPSRRRPSPPPRP
PSSSPRSRRPPPSPSSRPSRRRPPRPSRPSRSSSSPSSPPPPSPPRPSSSSRSSSPSSSPRRRPRRRRPPRRRPRSRPS
