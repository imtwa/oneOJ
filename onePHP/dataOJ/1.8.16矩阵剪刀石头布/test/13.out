SPPSRRPSPRSPRRPSSRSPRSRPRRRPSSRPSPRSSRSPPPRPSPRSPRRSPPRSPRPSSRPRPSRRPRSPPSPPRSS
RRRRPRSRSPRSPRSRPPRRRSSSPPPSRPPSRSPRRPRSSSRRRPSSRSPRRSPRSPSRRPSRRPPSSPRSPRPSSSR
PRRPSRSPRSPRSPSRPSRPPPRRPSSRPPSRPRSPPSPPRRPSSRPPSSRSPRSPRRPPPSPPPSRPRRRPSPPRRRP
SRPPSPSPSPRSPSRPSRPSSPPPSRSRSSSRSSPPSRPSPRRPPSSRSPRSPSPRSPPSRRPSRPRRSSRPRPSRPPS
RPSSRSRPRSPRSRPSPSRRRSSRPPRPRRSPPPSRSSRRSPRPRSPSSPSSRSRRSPRPPPSRSRPSSRPSPPSRSSR
SSRPPPRSSSPRRPSRSSRPPPPRPSRSSPRRPSPRSRRPRSPSPRSSRPPPSSRPPRSPSRRPSRSSPSPRRPRPRSP
PPRPSRSPPRSPPSRPSRRSSPRSSRPSSRSSRRSPRPSSPRPRPPSSRSPRRPPSPRSPSSSPPPSRSSRRSSSSPRS
RPSRPRSRPRSPRPSRRPPSRSPRRPSRSSSPSPRSRRPRRPSRSPRSPRSPRSPRSPSSRPPPSRPPPSRPSRPRSPP
SRPSPSRSSRSPRRSSSSRSPPPPSSPPPSRSSRSSPPPSPSRPSRSPPSSPSSRPRRRPRRRRPSRSPPRRPRSRSPR
RPSRPPPSPPRSPPSSRPRRRRSRPRRSSRPSPSSRSSRSPPRRPSSSRPRSPRSSRPRSPSSSRSSSPPRSRSRPSRS
PRPSSRRRPSRSSSRPPRPPSSPRPSRPRSPRRSSPRPSSRRSSPPRSSRSPRSSRPSRSSRRRRPSRPSPRRPSPSRS
SRPSRPSSRPPPRSSPSPSRSSRSPPSRPSRSPRRRSPPRPSPSRSPRPSSRRPSRSSPRSPPPPRRPSRPSRPSRSPP
PPSPSRPSRPSPRRSSRPSSSPSPSRSSRPRSRSPRSRRSRSPRSSRSRRPRSSRPSRSPPPSSSRSRPRRPSPPRPRR
PSRPSSRPRRRSPPRSSRRPRSPSSSSPSRRPSPRSSRPSRPRRPRSSSSPSSSSRRPSRSPRRRPSRPRSPPRRRRSS
PSRSRSSRSRPSRSPPPSSPSPSRRPRSPRSPPRSPPSRRPRSPSPPRRPRPRRPSSPPSSRSPPRPPSRSSRPSPRRS
RPPRPPRSPSRPRPRPRRPRPPSPRRSSRSPRRSPRPSRPSRRPRSPRSRSRRRPPPSRPRSSRSPPSRPPPRSPRRPS
RPSRSPSPRSSRSRSSRSSPPSRRSSSSRSPSRSPSSRRPSPRSPRSPRSRRPPSRSPSPRSRRPRRPSSPRSPRSSPR
SSPPPRPPRPPSPRPPPSRPSRPSSSRSPSSRPRSSRPSRRPSPPSPRSRPPSRSSSRSRSSPSPRSSRRPSPPPSRPS
SRSSPRRSPRRPPRRRRSSRSPRPRSPPRRSRSRRSSSRSSPSRRPPRPPSRPSPPPSPSPRRPRPSRPSRSRSPPSPP
RPSRSPPRSPRPSRPSSPSRPRSRSPRRRPRPRPPRRRPRSPPSSRSPPSRPRPSPRSPSRRRSPRPSRSRPRSRRPRR
RRRPSRSRSSRSPSPPSRSSPSPSPSRSSSPSRSSRPPSSPRPPRSPSSRPSPSRSPRSPRPPSPRPSSPSPSPSSRSS
PSRPSSSPRSPRSSPRPRPPRPRPRRPSRRSSPRRSSRRSPRSRSPPRRPSRRSPPSPPRPSRRSPSRPRPRPSRPSPP
RPPSRPRRRPRSSRSSRRPPRRSRPSRRPPSRSPPRRPPRSSSPPRSRPSRPSSSRRRRSPSRPPRPSPRRSPPSPPRP
SRRPRPSSPRSPRPRRPRSSPRSSPPRPSRRPSRSPRSSRPRRPPSRPSRPRPRSSPRSSPSRSPPRRPPSPSSRSPPS
SRRPSRSPRSSRPSPPSPRRSPPRRRPSSRSPPSPPSPPRRSSSPRRPSSSSPRPPRRSRSRPSRSPRSPPSRRPRSPR
RPSRPRPPSSRPSRPSRSSPSPRSPPSRPPSPSSPPSSRSPPPRRPPSRPRSPRRPSPRPSSPPRSPRSRRPPPSSPPR
SPPSRRRRPRSRSSRRPRSPRSPPRRPRRRRPRRSPRSSPPSPSPSRPRRSPRRSRSPSRRPPSSSPSSRRPSRRPRSS
PRRPSPSPPRRPRPSSSRSRSPPSRPPSSRPSPPSPSPRRRRSPPPSRSSPRSPPSPPSRSRSRPRRSPPRRRSSRRSS
RSRRPSRSSPPSPPPPPPRSSPRPSSRSPSRRRRRSPSPPSPRRSPRSPPPRSPPSPRPPSRPRRSRPPSRRPSPSPPR
SRPSRRPSRSSRSSPSRSSSRRPRPPRRPSRPRSPSRRPPSRSSSPSSRRSSPSSRSSRPSRPSRSSRRPPSPRRPRRS
PSSRSRPPPSRPSRPSRPRSPRSPSRRSPPPSRPSRPPSRPRPPRSPPRRPRRRPPRSPSSRPSPRSRPSRSRPRRSSP
PSRPSSSRPSPRSSPPRRSPPSSPPSPRRRRPSPPSSRPRRSRRRPRRSRSSPPPSRSSRPSPRSSPSPSRRPSRSPSR
RPSRRPPSSRSPPRRRRSPPSRRSPRSSSSRRPRPSRSPSPRSRSSRSRPRSSSRRPRRPSPPRSPPPRPPSRSPPPRS
PSRPSRSRPPSPPSPSPRRPSPPRSPPRSRPSPPSRPPRPRPSPRSPRSSRSRPPRSSRPRRSSSPSRSRPPRPRRSPP
SRPSRPSPPSRRRPPPRSSSRSSSPRSPRSRSRRPPSRRPSSRSSSRSSRPRSPSRPRSSPRSPRSPSRPSRSRRRPRR
PSPPSRRRRSPPPSSRSPSRPSRSSPPRPSRSSPRRPSSPSRPRSPSPPSSPRSPSRSSRPSPRRPRPSRPSPSPRSPS
RSRPSRPPSPRSRRPRSPRRPSPRRSSPRRPRRSPRRPPSRPSPRSPRRSRRRSPSSRPSRSPSRSPRPSSRRPRSPPR
SPRPPPSRPRSPSRPRRSPPSRRSPPRSPPSPPSSSSRSPSSRRRRSPSRPSPRSRSSRPSPRPPSPSPSRPRSPPRSS
SPPSPPSSPRSPPPSRPRSSRPSSRSPSPSRPPRRSSRPRPPPSPPRSPSPPPSPPRSSRSPRSRRSPRPPSPPRSPRR
RPSRPSRPRSSRSPPSSRRRPPPRSPRRSSRSSRPRRSRRRPSRPSPRSSRRRPPSSRRPRSSSRPRRPSPRSPRRRSP
PRPSPPSRSPPSPPPSRPPSSSRSRSRPSRPSRPSPPSRPSPPSSRPSPPSPRSPRSPRSSPSRPSRPSRPRSRSPRSP
SPPPPRPSPPSRPSSRPSRPRRRPPRSRSSRSSRRPPSSRPRRPPSRSPRPRRPRSSRPRSSRPPSPPPRSPPSPRSSR
RPSRSSRPRSRPPRSSRPRPSPRSSSSSPRSSRPPSPRSRRRSRPPRPRRRSSRSSRSSRRSPSRRPSPRSPRPRSPRS
SPPRSSRRRPSSSSPPRRRPPRPSRPRSPRSRPSPRRSRPSPPSSRRPSPRSRPRSPRSPPRRRPSPRPSPRRRRSSPP
PSRSSRPSPPSRRPRRRSSSRSRPSPSPPSPSRRPSSPSRPRRPPSSRSRSPSSRPRSSRSPRSRSPRRPRRPSPPRSP
RPSPRSSRPSRPSPRRPSRPSPSRSRPRPSRPPSPPPRSSPPSPRPSRPRPRPPSPRPPRSRSPSPPSSPRSRPRSSPR
